lib/publish.js

```diff
@@ -149,7 +149,8 @@ export default async (pluginConfig, context) => {
               throw error;
             }
 
-            const { url, alt } = response;
+            const { alt } = response;
+            const url = urlJoin(gitlabUrl, response["full_path"]);
```

Comment:
This breaks the unit tests. Can you locally run `npm run test`.

```suggestion
            const url = urlJoin(gitlabUrl, response.full_path);
```

In `ln. 177` we are already building the url for the assets using `urlJoin(gitlabUrl, repoId, url)`.

Reply:
see https://github.com/semantic-release/gitlab/pull/754/commits/1c48f54e47ffd47fba3fa172e77c75f7fc4f738b#diff-c8bd325917d181608f40adecdcec3c76acde134cc15a9e4a0d18636698e4fe55R152-R153

matching the previously used style